lib/mime.c

```diff
@@ -722,7 +722,6 @@ static size_t readback_bytes(mime_state *state,
   sz = numbytes - state->offset;
 
   if(numbytes > state->offset) {
-    sz = numbytes - state->offset;
```

Comment:
I don't see how this one is right. sz is used later on

Reply:
nm I see it now, it was set above. it would probably be easier to understand if it was set here and the one above removed

Reply:
Yeah probably, was more thinking that it could benefit of being set unconditionally first so that one wouldn't think it risked ending up not being set...